apps/api/models.py

```diff
@@ -102,24 +105,50 @@ class APIKey(models.Model):
     class Meta:
         db_table = 'api_key'
 
+    def __unicode__(self):
+        return (
+            u'<{cls} user={user}, type={type}, key={key} secret=...>'
+            .format(cls=self.__class__.__name__, key=self.key,
+                    type=self.type, user=self.user))
+
     @classmethod
     def get_jwt_key(cls, **query):
```

Comment:
Not related to this bug, but presumably this requires a user object in the query? Otherwise some could do `get_jwt_key()` and get someone elses key. Although admittedly that would require there to be one and only one jwt key in existance to avoid a multiple object error. Anyway I like that `new_jwt_credentials` requires a user, thinking this API probably could too.